cardscanner/src/main/java/company/tap/cardscanner/TapCountDownTimer.java

```diff
@@ -0,0 +1,42 @@
+package company.tap.cardscanner;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.CountDownTimer;
+import android.view.View;
+import android.widget.Toast;
+
+import java.net.URISyntaxException;
+
+
+/**
+ * Created by AhlaamK on 4/2/20.
+ * <p>
+ * Copyright (c) 2020    Tap Payments.
+ * All rights reserved.
+ **/
+public class TapCountDownTimer extends CountDownTimer {
```

Comment:
Why we have TapCountDownTimer, it's useless now

Reply:
resolved